tests/nfs2-frames/test.rules

```diff
@@ -0,0 +1,3 @@
+alert nfs any any -> any any (flow:to_server; frame:rpc.creds; content:"|00 00 00 09|werrmsche|00 00 00|"; sid:1;)
```

Comment:
with all these rules it would be good to consider anchoring them to the start and end of buffers with `startswith`, `endswith` and/or `depth`/`offset`. This way the buffers are checked more exact.